proforma.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/openHPI/proforma'
   spec.license       = 'MIT'
 
+  spec.required_ruby_version = '~> 2.6'
```

Comment:
Do we have any specific reason for 2.6? As this is forward-compatible, 2.7 shouldn't be a problem, I assume? What about 3.0?

Reply:
I'm not sure about 3.0 - I'd check it once we can start using that for the rails apps

Reply:
Alright, sounds like a good plan. Let's see when we will be ready to use Ruby 3.0